tooling/scripts/validate-dependencies.groovy

```diff
@@ -24,7 +24,7 @@ final File pomXml = new File(project.basedir, "pom.xml")
 /* groupIds that contain extensions */
 final Set<String> extensionGroupIds = ["org.apache.camel.quarkus", "io.quarkus"] as Set
 /* artifactIds from groups contained in extensionGroupIds that are not extensions */
-final Set<String> nonExtensionArtifactIds = ["quarkus-development-mode-spi"] as Set
+final Set<String> nonExtensionArtifactIds = ["quarkus-development-mode-spi", "quarkus-qute"] as Set
```

Comment:
Why this change? `quarkus-qute` is an extension _par excellence_

Why rather not adding `io.quarkus:quarkus-qute-deployment` to your deployment module to get rid of the related error message?

Reply:
OK, I will try to add it. Thanks !

Reply:
Great and it works.